src/exo/API_scheduling.py

```diff
@@ -1788,6 +2052,11 @@ def reorder_loops(proc, nested_loops):
                           variable of the inner loop.  An optional '#int'
                           can be added to the end of this shorthand to
                           specify which match you want,
+        rc              - (bool) whether to return relevant cursors
+
+    relevant cursors:
+        outer_loop      - a cursor to the new outer loop
+        inner_loop      - a cursor to the new inner loop
```

Comment:
Should these (and elsewhere) be `outer` and `inner` in keeping with your stated preference for shorter names?